src/libs/actions/ReimbursementAccount/setupWithdrawalAccount.js

```diff
@@ -317,7 +245,8 @@ function setupWithdrawalAccount(params) {
             }
 
             // Go to next step
-            navigation.goToWithdrawalAccountSetupStep(getNextStep(updatedACHData), responseACHData);
+            // Note: php sometimes returns code 200 with no achData, just to indicate that we should keep whatever the JS values are.
```

Comment:
Contributors who work on this code might not connect that when we refer to "php" we mean the API (though it's not hard to figure out from the context).

Reply:
NAB, this note seems important, but I had some trouble understanding when it could happen. Is it important to know not just that it can happen but when? 

Anyway, not sure if this is really valuable but maybe we can log something instead of leave this comment

```js
if (_.isEmpty(responseACHData)) {
    Log.info('[SetupWithdrawalAccount] No achData in response. Navigating to next step based on currently stored values.');
}
```

Reply:
Some steps don't need to verify or save anything (like [here](https://github.com/Expensify/Web-Secure/blob/main/lib/BankAccountAPI.php#L1089)), and we early return, just to say that the JS needs to go to the next step with whatever data they already have.